phoenix-core/src/main/java/org/apache/phoenix/compile/FromCompiler.java

```diff
@@ -858,7 +868,7 @@ public ColumnRef resolveColumn(String schemaName, String tableName, String colNa
                 while (iterator.hasNext()) {
                     TableRef tableRef = iterator.next();
                     try {
-                        PColumn column = tableRef.getTable().getColumn(colName);
+                        PColumn column = tableRef.getTable().getPColumnForColumnName(colName);
```

Comment:
I see why you renamed it. How about getColumnFromName() and getColumnFromQualifier()?